source/fundamentals/crud/write-operations/change-a-document.txt

```diff
@@ -0,0 +1,183 @@
+   :dedent:
+   :start-after: begin updateManyExample
+   :end-before: end updateManyExample
+
+The method returns:
+
+.. code-block:: json   
+
+    AcknowledgedUpdateResult{ matchedCount=5, modifiedCount=5, upsertedId=null } 
+
+The following shows the updated documents in the ``paint_inventory`` collection:
+
+.. code-block:: json 
+
+    { "_id": 1, "color": "red", "qty": 25 }
+    { "_id": 2, "color": "purple", "qty": 28 }
+    { "_id": 3, "color": "yellow", "qty": 20 }
+    { "_id": 4, "color": "green", "qty": 26 }
+    { "_id": 5, "color": "pink", "qty": 20 }
+
+If zero documents matched the query filter in the update operation,
+there are no changes to the collection. See our :doc:`upsert
+page</fundamentals/crud/write-operations/upsert>` to learn how you can
+change the collection with zero documents matched. 
+
+.. note::
+
+    You cannot modify the unique index of a document. See the MongoDB
+    server manual for more information on :manual:`unique indexes
+    </core/index-unique/>`.
+
+.. _replace-operation:
+
+Replace
+-------
+
+A replace operation substitutes one document that matches your query
+filter with a replacement document using the
+:java-docs:`replaceOne() <apidocs/mongodb-driver-sync/com/mongodb/client/MongoCollection.html#replaceOne(org.bson.conversions.Bson,TDocument)>` 
+method.
+
+Replace Operation Parameters:
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- Query filter: specifies the document you want to replace in your collection
+- Replacement document: a new ``Document`` with the fields and values to replace in the document matched
+
+The ``repalceOne()`` method uses the following format: 
+
+.. code-block:: java
+
+    collection.replaceOne(<query filter>, <new Document>);
+
+Example
+~~~~~~~
+
+The paint store realizes they need to update their inventory again.  What they
+thought was 20 cans of pink paint is actually 25 cans of orange paint. 
+
+To update the inventory, call the ``replaceOne()`` method specifying the
+following:
+
+- A query filter that matches the ``color`` is "pink" in the ``paint_inventory`` collection
+- A replacement document that contains: ``color`` is "orange" and ``qty`` is "25"
+
+.. literalinclude:: /includes/fundamentals/code-snippets/Update.java
+   :language: java
+   :dedent:
+   :start-after: begin replaceOneExample
+   :end-before: end replaceOneExample
+
+The method returns:
+
+.. code-block:: json   
+
+    AcknowledgedUpdateResult{ matchedCount=1, modifiedCount=1, upsertedId=null }
+
+The following shows the updated document:
+
+.. code-block:: json
+
+    { "_id": 5, "color": "orange", "qty": 25 }
+
+If zero documents matched the query filter in the replace operation,
+there are no changes to the collection. See our :doc:`upsert
+page</fundamentals/crud/write-operations/upsert>` to learn how you can
+change the collection with zero documents matched. 
+
+If multiple documents match the ``replaceOne()`` method, the method
+replaces the first matched document matched by the query. 
```

Comment:
I think the subject, verb, and objects of this sentence could be clarified by reordering the sentence..

```suggestion
If multiple documents match the query filter specified in the``replaceOne()`` method, it replaces the first result.
```